### PR TITLE
getting rid of littledata:synced-cron #214

### PR DESCRIPTION
**Pull request by pozylon**

### CODE REVIEW

File: packages/platform/setup-workqueue.js [hunk 1]
```diff
@@ -20,14 +19,9 @@ export default ({ cronText }) => {
...
-  //   new IntervalWorker({
-  //     WorkerDirector,
-  //   })
-  // );
   if (cronText) {
     handlers.push(
-      new CronWorker({
+      new IntervalWorker({
         WorkerDirector,
         cronText,
```
Comment (Mikearaya) [inline packages/platform/setup-workqueue.js:26]: Has the format change, is it Linux corn text now.

**Review by Mikearaya — APPROVED** [review]
I see you activated the interval work. looks good to me

File: packages/platform/setup-workqueue.js (hunk 1 above)
Comment (pozylon) [reply packages/platform/setup-workqueue.js:26]: No I think it's the same like before, isn't it?

Comment (pozylon) [reply packages/platform/setup-workqueue.js:26]: You're right it's actually not cron it's kind of a text parser (http://bunkat.github.io/later/parsers.html#text) so the name is missleading, we should support both

Comment (pozylon) [reply packages/platform/setup-workqueue.js:26]: I saw we have this problem at different places there is even a setting "autoSchedulingCronText" that is actually not cron it's later.js text expression.

**Comment by pozylon** [comment]
@Mikearaya I took a radical approach and removed cronText completely, the name was so missleading and deprecating it would not have helped there.

**Comment by pozylon** [comment]
At least now you could use linux cron tab expressions for the interval worker (https://bunkat.github.io/later/parsers.html#cron)

**Pull request merged**